katas/langtonant/maria/antolangt.js

```diff
@@ -0,0 +1,180 @@
+
+function Grid(n, m) {
```

Comment:
maybe this can be converted to a class since it has different functions that could be considered its functionality